docs/services/slam/cartographer/_index.md

```diff
@@ -0,0 +1,364 @@
+---
+title: "Cartographer Integrated Library"
+linkTitle: "cartographer"
+weight: 70
+type: "docs"
+description: "Configure a SLAM service with the Cartographer library."
+tags: ["slam", "services"]
+# SMEs: Kat, Jeremy
+---
+
+[The Cartographer Project](https://github.com/cartographer-project) performs dense SLAM using LIDAR data.
+
+## Requirements
+
+Install the binary required to utilize the `cartographer` library on your machine and make it executable by running the following commands according to your machine's architecture:
+
+{{< tabs >}}
+{{% tab name="Linux aarch64" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+sudo curl -o /usr/local/bin/carto_grpc_server http://packages.viam.com/apps/slam-servers/carto_grpc_server-stable-aarch64.AppImage
+sudo chmod a+rx /usr/local/bin/carto_grpc_server
+```
+
+{{% /tab %}}
+{{% tab name="Linux x86_64" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+sudo curl -o /usr/local/bin/carto_grpc_server http://packages.viam.com/apps/slam-servers/carto_grpc_server-stable-x86_64.AppImage
+sudo chmod a+rx /usr/local/bin/carto_grpc_server
+```
+
+{{% /tab %}}
+{{% tab name="MacOS" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+brew tap viamrobotics/brews && brew install carto-grpc-server
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Configuration
+
+How you configure `cartographer` depends on whether you want the SLAM service to build your map with data collected live by a [RPLIDAR](https://www.slamtec.com/en/Lidar/A3) or with LIDAR data provided in a dataset at runtime.
+
+Select from the following modes to obtain the correct instructions to configure the service:
+
+{{% tabs name="Modes"%}}
+{{% tab name="Live Data Collection" %}}
+
+{{% alert title="REQUIREMENTS" color="tip" %}}
+
+Running `cartographer` in Live Data Collection mode requires a [RPLIDAR A1](https://www.slamtec.com/en/Lidar/A1) or [RPLIDAR A3](https://www.slamtec.com/en/Lidar/A3) LIDAR scanning device.
+
+Before adding a SLAM service, you must follow [these instructions](/program/extend/modular-resources/add-rplidar-module/) to add your RPLIDAR device as a modular component of your robot.
+
+{{% /alert %}}
+
+<br>
+
+### Add a SLAM Service
+
+{{< tabs name="Add the Cartographer Service Live">}}
+{{% tab name="Config Builder" %}}
+
+Go to your robot's page on the [Viam app](https://app.viam.com/).
+Navigate to the **config** tab on your robot's page, and click on the **Services** subtab.
+
+Add a service with type `slam`, model `cartographer`, and a name of your choice:
+
+![adding cartographer slam service](../img/run_slam/add-cartographer-service-ui.png)
```

Comment:
would you mind retaking this screenshot and making the window less wide before taking it so it's more legible on small width screens?

Reply:
done!